wizeline/remix-project-lab
# Feature/135/profile view page

# What does this PR do?

Developed the user profile component, where the user can see their information and past projects. This information will come from GitHub.

Made the call to get the information from GitHub, built a table to store the information obtained, stored the information in the table, queried and displayed the information on the frontend.


# Where should the reviewer start?

Run the project locally and select the profile option by clicking your email in the home page navigation bar.


# How should this be manually tested?

- Go to the profile page and check if your GitHub information is displayed in a responsive way
	- Username
	- Projects


# What are the relevant tickets?

135, 150 and 153


# Screenshots

![image](https://github.com/wizeline/remix-project-lab/assets/42613217/e30b6c14-ec69-40d1-a416-443734ed18d4)

![image](https://github.com/wizeline/remix-project-lab/assets/42613217/37bf20bb-b117-4077-b626-60ca7b1715b9)

![image](https://github.com/wizeline/remix-project-lab/assets/42613217/93516fcf-8fb0-4c30-8b5a-b24212b561ea)


# Checklist
